dependencies.gradle

```diff
@@ -53,8 +53,8 @@ libraries["hibernate"] = "org.hibernate.javax.persistence:hibernate-jpa-2.1-api:
 libraries["hibernateC3p0"] = "org.hibernate:hibernate-c3p0:5.6.7.Final"
 libraries["hibernateJcache"] = "org.hibernate:hibernate-jcache:5.6.7.Final"
 libraries["hibernateJpamodelgen"] = "org.hibernate:hibernate-jpamodelgen:5.6.12.Final"
-libraries["hibernateValidator"] = "org.hibernate:hibernate-validator:5.4.3.Final"
-libraries["hibernateValidatorProcessor"] = "org.hibernate:hibernate-validator-annotation-processor:5.4.3.Final"
+libraries["hibernateValidator"] = "org.hibernate:hibernate-validator:6.2.5.Final"
+libraries["hibernateValidatorProcessor"] = "org.hibernate:hibernate-validator-annotation-processor:6.2.5.Final"
```

Comment:
Let's also update the namespace from `org.hibernate:` to `org.hibernate.validator:`, because apparently they have moved to that one (as per the note [here](https://mvnrepository.com/artifact/org.hibernate/hibernate-validator)).

Reply:
Good catch! I missed that note when I was looking up the versions